test/bench_package.cpp

```diff
@@ -333,11 +333,11 @@ BENCHMARK(BM_MxV_CX_ControlBottom_TargetTop)->Apply(QubitRange);
 static void BM_MxV_HadamardLayer(benchmark::State& state) {
     unsigned short nqubits = state.range(0);
     auto           dd      = std::make_unique<dd::Package>();
```

Comment:
this PR is for dynamic qubit size and yet the benchmarks still utilize a static qubit size instead of the new ability to set the qubit count at construction time.